resources/README.md

```diff
@@ -0,0 +1,170 @@
+Table of Contents
+=================
+
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
+- [Overview](#overview)
+  - [Validation](#validation)
+  - [JSON-schemas](#json-schemas)
+  - [Versioned Store Write Operations](#versioned-store-write-operations)
+  - [Validation schemas](#validation-schemas)
+- [Workflow](#workflow)
+- [Properties](#properties)
+  - [Property Types](#property-types)
+<!-- TOC END -->
+
+# Overview
+
+This section is meant to complement the explanation of the concepts on the repo [landing readme](../README.md)
+The design of the system and its workflow is outlined below. For some, it may be easier to follow the [examples](examples) instead, or read them in parallel.
+
+## Validation
+Although the actual extrinsics and runtime operations happens through the two [substrate](https://github.com/paritytech/substrate) modules [substrate-versioned-store-module](https://github.com/Joystream/substrate-versioned-store-module) and [substrate-versioned-store-permissions-module](https://github.com/Joystream/substrate-versioned-store-permissions-module), both the `Content Curators` and `Content Creators` will talk to these modules through the [versioned-store-js](https://github.com/Joystream/versioned-store-js).
+
+## JSON-schemas
+This [versioned-store-js](https://github.com/Joystream/versioned-store-js) tool allows us to use "human readable" `JSON-schemas`, which must not be confused with [schemas](../README.md#schemas) or [validation JSON-schemas](#validation-schemas).
+
+The advantage of using `JSON-schemas` that follows the [JSON Schema standard](https://json-schema.org/) is that it's both structured enough to avoid confusion, yet far easier to learn and parse than `rust` code. The `JSON-schemas` can be used to perform all 6 [operations](#versioned-store-write-operations) as the [versioned-store-js](https://github.com/Joystream/versioned-store-js) will use them as input, validate them, construct the `extrinsics`, and broadcast them.
```

Comment:
```suggestion
The advantage of using `JSON-schemas` that follows the [JSON Schema standard](https://json-schema.org/) is that they are both structured enough to avoid confusion, yet far easier to learn and parse than `rust` code. The `JSON-schemas` can be used to perform all 6 [operations](#versioned-store-write-operations) as the [versioned-store-js](https://github.com/Joystream/versioned-store-js) will use them as input, validate them, construct the `extrinsics`, and broadcast them.
```